src/extension.ts

```diff
@@ -29,6 +30,14 @@ export function activate(context: vscode.ExtensionContext): void {
         config.path = context.asAbsolutePath(join('vendor', 'phpactor', 'phpactor', 'bin', 'phpactor'))
     }
 
+    if (!config.executablePath) {
+        const phpConfig = vscode.workspace.getConfiguration('php')
+        config.executablePath =
+            phpConfig.get<string>('executablePath') ||
+            phpConfig.get<string>('validate.executablePath') ||
```

Comment:
`validate.executablePath`?

Reply:
php.executablePath and php.validate.executablePath are used as fallbacks and are "inherited" from Microsofts internal PHP tooling as well as vscode-php-debug and vscode-php-intellisense. https://code.visualstudio.com/docs/languages/php#_linting